rest-express-mongo/README.md

```diff
@@ -14,26 +14,133 @@ The QuickWrites is a simple web application built with Express.js for managing n
 - Node.js installed on your machine
 - MongoDB Atlas account or a local MongoDB database.
 
-## Start the server:
+## Install Keploy
+Install keploy via one-click:-
 
-`npm start`
+```bash
+curl --silent -O -L https://keploy.io/install.sh && source install.sh
+```
 
+## Setup application
+Clone the repository and move to express-mongo folder
+```bash
+git clone https://github.com/keploy/samples-typescript && cd samples-typescript/rest-express-mongo
+
+# Install the dependencies
+npm install
+```
 ## Running test cases in Keploy:
 
-`keploy test -c "CMD_TO_RUN_APP" --delay 10`
+`keploy record -c "npm start"`
+
+We will get the following output in the terminal
+
+![Testcase](./Img/keploy.png)
+
+#### Let's generate the testcases.
+
+Make API Calls using [Hoppscotch](https://hoppscotch.io), [Postman](https://postman.com) or cURL command. Keploy with capture those calls to generate the test-suites containing testcases and data mocks.
+
+```bash
+curl --request POST \
+  --url http://localhost:3000/ \
+  --header 'content-type: application/json' \
+  --data '{
+    "title": "Sample Note",
+    "body": "This is the body of the note",
+    "username": "testuser"
+}'
+```
+
+🎉 Easy right! Just one API call and you've whipped up a test case with a mock. Check out the Keploy directory to find your shiny new `test-1.yml` and `mocks.yml` files.
+
+## Run keploy test
+
+Want to see it in action? Run the following command to execute your Keploy tests
+
+```bash
+keploy test -c "npm run" --delay 10
+```
+
+Great job following along 🥳! Now, let's dive deeper and explore how to do Keploy integration with jest test 📌
+
+## Get Keploy jest sdk
+
+```bash
+npm i @keploy/sdk nyc jest
+```
+
+## Update package file
+
+Update the `package.json` file that runs the application:
+
+```json
+ "scripts": {
+    //other scripts
+    "test": "jest --coverage --collectCoverageFrom='src/**/*.{js,jsx}'",
+    "coverage": "nyc npm test && npm run coverage:merge && npm run coverage:report",
+    "coverage:merge": "mkdir -p ./coverage && nyc merge ./coverage .nyc_output/out.json",
+    "coverage:report": "nyc report --reporter=lcov --reporter=text"
+    //other scripts
+  }
+```
 
 ## Usage
```

Comment:
Keploy test output is missing. Also we are moving towards no sdk integration approach. Therefore the keploy.test.js file is not anymore required.